pkg/kudoctl/cmd/plan/plan_history.go

```diff
@@ -62,10 +61,10 @@ func planHistory(args []string, options *Options, settings *env.Settings) error
 
 	var labelSelector string
 	if len(args) == 0 {
-		fmt.Printf("History of all plan-executions for instance \"%s\" in namespace \"%s\":\n", options.Instance, options.Namespace)
+		fmt.Printf("History of all plan executions for instance \"%s\" in namespace \"%s\":\n", options.Instance, options.Namespace)
 		labelSelector = fmt.Sprintf("instance=%s", options.Instance)
 	} else {
-		fmt.Printf("History of plan-executions for instance \"%s\" in namespace \"%s\" to operator-version \"%s\":\n", options.Instance, options.Namespace, args[0])
+		fmt.Printf("History of plan-executions for instance \"%s\" in namespace \"%s\" for operator-version \"%s\":\n", options.Instance, options.Namespace, args[0])
```

Comment:
```suggestion
		fmt.Printf("History of plan executions for instance \"%s\" in namespace \"%s\" for operator-version \"%s\":\n", options.Instance, options.Namespace, args[0])
```

Reply:
Looking at plan history cmd more... I'm not sure any of this makes sense.  you mentioned in our walk thru that this is slated for the next release... which I completely support.   I wonder if we should just remove these commands or stub them out.   they will not be functional.

the following:
```
	planExecutionsGVR := schema.GroupVersionResource{
		Group:    "kudo.dev",
		Version:  "v1alpha1",
		Resource: "planexecutions",
	}
```
is not functional